src/core_modules/capture-core/components/WidgetEnrollmentEventNew/Validated/useLifecycle.js

```diff
@@ -96,9 +96,12 @@ export const useLifecycle = ({
     ]);
 
     const rulesReady =
+    (!orgUnit) ||
```

Comment:
Isn't the orgUnit mandatory for the rules engine? Should we use `&&` instead of `||`?